docs/manuals/cli/run.md

```diff
@@ -0,0 +1,118 @@
+# Run application in Taipy CLI
+
+Assume that you have a "main.py" file containing your Taipy application.
+To start the Taipy application, you can run:
+
+```console
+$ taipy run main.py
+```
+
+!!! note
+
+    An alternative way to run your application is to use the Python command directly.
```

Comment:
"use the Python command" => "use the \`python\` command"
"command directly" => "command, which runs the Python interpreter."

Reply:
```suggestion
    An alternative way to run your application is to use the `python` command, which runs the Python interpreter.
```